### DISCUSSION TITLE
can you let me turn off project files?

### DISCUSSION THREAD

**Discussion by IonizedBanana** [mifi/lossless-cut#1187] [Ideas]
title, I only use this because I record my clips in 1080p60 and mp4 has terrible quality, so I use mkv. The default windows trim tool always outputs in mp4, which makes the framerate and quality decrease. I use this so i can still use mkv files, and i dont plan on ever opening the project again, im just trimming them. Please let me turn off project files, or at least let me set a separate directory somewhere in my documents folder so i dont have to delete them every time i trim a clip. thanks

**Comment by mifi** [comment]
does the "Auto save project file?" setting not work?

**Reply by IonizedBanana** [reply]
oh my god im so stupid, thanks lmao i completely missed that option
